draft-ietf-quic-http.md

```diff
@@ -384,11 +385,12 @@ client MUST treat this as a connection error of type HTTP_DUPLICATE_PUSH.
 
 ### Reserved Stream Types {#stream-grease}
 
-Stream types of the format `0x1f * N` are reserved to exercise the requirement
-that unknown types be ignored. These streams have no semantic meaning, and can
-be sent when application-layer padding is desired.  They MAY also be sent on
-connections where no request data is currently being transferred. Endpoints MUST
-NOT consider these streams to have any meaning upon receipt.
+Stream types of the format `0x1f * N` for non-zero values of N are reserved to
+exercise the requirement that unknown types be ignored. These streams have no
+semantic meaning, and can be sent when application-layer padding is desired.
```

Comment:
"Semantic meaning" sounds like a tautology.

(This formulation is not introduced in this change, but I figured I'd highlight it while I am at it.)

Reply:
Changed to "semantics" both places this occurs.